source/_posts/2017-11-07-which-hass-to-use.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Which installation should I do"
```

Comment:
Suggest another title: Which installation method should I use?

Reply:
Agreed, I was tired and had had a few drinks. :)